posts/2023-11-24_wyg-dongwon-shin/README.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Who You Gno – On the Record with Dongwon Shin"
+publication_date: 2023-11-24T00:00:00Z
+slug: wyg-dongwon-shin
+tags: [whoyougno, onbloc, community, interview]
+authors: [christina]
+---
+
+# Who You Gno – On the Record with Dongwon Shin
+*Who You Gno is intended to shine a light on the builders, contributors, and generally brilliant humans behind the tech. We’re excited to kick off this series with Dongwon Shin, the founder of one of Gno.land’s longest-contributing teams, Onbloc.*
```

Comment:
*Who You Gno is intended to shine a light on the builders, contributors, and generally brilliant humans behind the tech. We’re excited to kick off this series with Dongwon Shin, the founder of one of Gno.land’s longest-contributing teams, Onbloc.*
```suggestion
*Who You Gno is intended to shine a light on the builders, contributors, and generally brilliant humans behind the tech. We’re excited to kick off this series with Dongwon Shin, the founder of one of Gno.land’s longest-contributing teams, Onbloc.*

```

Reply:
Add spacing for readability

Reply:
Fixed with [e6bff10](https://github.com/gnolang/blog/pull/51/commits/e6bff10fd24dda48404c90c9ff495cd9d33a74b1).